packages/x-charts/src/ChartsTooltip/ChartsTooltip.tsx

```diff
@@ -2,9 +2,10 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 import composeClasses from '@mui/utils/composeClasses';
 import { styled, useThemeProps } from '@mui/material/styles';
-import { Popper, PopperProps } from '@mui/base/Popper';
+import { Popper, PopperProps as BasePopperProps } from '@mui/base/Popper';
 import { NoSsr } from '@mui/base/NoSsr';
 import { useSlotProps } from '@mui/base/utils';
+import { SxProps, Theme } from '@mui/system';
```

Comment:
```suggestion
import { SxProps, Theme } from '@mui/material/styles';
```

Looks like other elements are using the `@mui/material/styles`

I would be in favor of doing the same for consistency across types. With the material style the `axisContent` slot get auto completion.

![image](https://github.com/mui/mui-x/assets/45398769/bf711bda-a8ec-4afe-9b30-79cd3454d979)

Reply:
Fixed